why-polars-for-data-cleaning/index.qmd

```diff
@@ -0,0 +1,202 @@
+-   [Polars](https://pola.rs/)
+
+### DuckDB
+
+DuckDB is an in-process online analytical process (OLAP) database tool
+that has API extensions in many languages including Python. It is a very
+young tool, version 1.0.0 was released in 2024, although the first
+version was released in 2019. The developers have set it up as a
+not-for-profit foundation in the Netherlands.
+
+::: columns
+::: column
+#### Benefits
+
+-   Although a very young project it is already in use by a number of
+    large companies and has a large community of users.
+    -   The code is open source, and the way it has been set up is
+        promising for it to remain that way.
+-   It is very fast, and can handle large data sets.
+-   It is designed for data cleaning using SQL syntax.
+-   Extensive documentation with user guide sections embedded.
+:::
+
+::: column
+#### Drawbacks
+
+-   It is not designed exclusively to be used as a data cleaning tool,
+    this is more of an add-on on the way to storing data.
+-   To use it to its full potential, users need to have some knowledge
+    of SQL.
+:::
+:::
+
+### Polars
+
+The Polars project started in 2020 and is based in the Netherlands. They
+have a relatively small team but seem to have a fast growing user base,
+and very enthusiastic developers. Polars is written in Rust and
+available for Python. Because it is written in Rust (which is a low
+level language), it is very fast, and can handle all common data
+formats. The two main developers are described as enthusiastic about
+open source.
+
+::: columns
+::: column
+#### Benefits
+
+-   Enthusiastic user community, with a lot of support available.
+-   Supports several languages, including Python.
+-   Handles all common data formats.
+-   Faster than Pandas.
+-   Polars have an extensive user guide section.
+:::
+
+::: column
+#### Drawbacks
+
+-   Relatively young project, so may not have all the features of more
+    established tools.
+-   Because it is still new, it isn't as widely used and doesn't have
+    too many learning resources available yet.
+-   No traditional documentation section, everything is in the user
+    guide.
+:::
+:::
+
+### Pandas
+
+Pandas was initially released in 2008, and is now a well established
+tool to work with data frames in Python. The tool is written in Python,
+and is designed to be easy to use.
+
+::: columns
+::: column
+#### Benefits
+
+-   Established tool with a large user base.
+-   Well supported, and well documented.
+-   Plenty of learning resources available from a variety of sources.
+-   Works with all relevant languages and data formats.
+-   The pandas website contains a very thorough section on how to get
+    started, including customised tutorials for people that have
+    experience with many other tools.
+-   The documentation seem very thorough, and includes a lot of
+    examples.
+:::
+
+::: column
+#### Drawbacks
+
+-   Can be slow when working with large data sets.
```

Comment:
```suggestion
-   Can be slow when working with large data sets.
-   Because it was developed more than a decade ago, some of it's syntax and design is out-dated.
```